### PR TITLE
Updated and expanded the social media section (#49, #29)

### PR DESCRIPTION
**Pull request by AppleC4T**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`
- Preview Link
Before changes
before
After changes
post

## Description / Changes
Updated the social media section in the footer by adding more icons linking to the following social media: facebook, youtube, twitter, instagram, github, fossasia gci2018 organization profile, fossasia main website and linking them to the actual sites, so that the links work properly.

### CODE REVIEW

**Comment by AppleC4T** [comment]
@tabesin check if everything is ok now, I added back the CNAME file.

**Comment by tabesin** [comment]
Maybe put it in your local .gitignore

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-gci18.fossasia.org
```
Comment (liveHarshit) [inline CNAME]: Why these changes?  Revert it.

Comment (AppleC4T) [reply CNAME]: It's already reverted and the CNAME content is brought back. I accidentally changed it while testing my changes.

**Comment by liveHarshit** [comment]
Please squash the commit asap otherwise it will conflict with another commit.

**Comment by liveHarshit** [comment]
> @liveHarshit done.

It is still showing `This branch has conflicts that must be resolved`

**Comment by AppleC4T** [comment]
@liveHarshit check now. The conflicts should have been resolved.

**Comment by liveHarshit** [comment]
Not resolved. `This branch cannot be rebased due to conflicts`

**Comment by AppleC4T** [comment]
@liveHarshit for me it says 

> This branch has no conflicts with the base branch

>
